src/mocks/data/reportCounts.ts

```diff
@@ -3,28 +3,28 @@ import { ReportCounts } from "src/utils/api/reportCounts";
 import { FileSubmissions, FileSubmission } from "src/utils/api/fileSubmissions";
 
 export const generateCounts = (submissions: FileSubmissions): ReportCounts => {
-  const total = submissions.summary.total_items;
-  const completed = submissions.items.filter((el: FileSubmission) =>
-    el.status.toLowerCase().includes("complete")
+  const total = submissions.summary.totalItems;
+  const completed = submissions.items.filter(
+    (el: FileSubmission) => el.status == "DELIVERED"
   );
-  const failed = submissions.items.filter((el: FileSubmission) =>
-    el.status.toLowerCase().includes("failed")
+  const failed = submissions.items.filter(
+    (el: FileSubmission) => el.status == "FAILED"
   );
-  const processing = submissions.items.filter((el: FileSubmission) =>
-    el.status.toLowerCase().includes("uploading")
+  const processing = submissions.items.filter(
+    (el: FileSubmission) => el.status == "PROCESSING"
   );
 
   return {
     total_counts: total,
```

Comment:
Is this the structure that PS API will return?  I thought it was grouped by report type instead of status type.

Reply:
Nope, here is the counts return type: 

```
{
  "data": {
    "processingCounts": {
      "statusCounts": {
        "failed": {
          "counts": 20,
          "reasons": {
            "metadata": 0
          }
        },
        "delivered": {
          "counts": 320,
          "reasons": null
        },
        "processing": {
          "counts": 2,
          "reasons": null
        }
      },
      "totalCounts": 342
    }
  }
}
```